src/resources/views/crud/list.blade.php

```diff
@@ -51,7 +51,7 @@
 
         {{-- Backpack List Filters --}}
         @if ($crud->filtersEnabled())
-          @include('crud::inc.filters_navbar')
+          <x-backpack::filters-navbar :crud="$crud"/>
```

Comment:
Does this blade file really need the `$crud`... or does it actually just require the `$filters`? Cause in the blade file the only mention is `$crud->filters`. Don't know which one would be better to pass... but i'm leaning towards `$filters`... Don't have a strong opinion on this so feel free to ignore it.